api_tests/registrations/views/test_withdrawn_registrations.py

```diff
@@ -191,3 +192,14 @@ def test_field_specific_related_counts_retrieved_if_visible_field_on_withdrawn_r
         res = app.get(url, auth=user.auth)
         assert res.status_code == 200
         assert res.json['data']['relationships']['contributors']['links']['related']['meta']['count'] == 1
+
+    def test_child_inherits_withdrawl_justication_and_date_withdrawn(
+            self, app, user, registration, withdrawn_registration):
+        child_id = withdrawn_registration.registrations.first()._id
+
+        url = '/{}registrations/{}/'.format(API_BASE, child_id)
+        res = app.get(url, auth=user.auth)
+        assert res.status_code == 200
+        assert res.json['data']['attributes']['withdrawal_justification'] == withdrawn_registration.justification
+        formatted_date_retracted = withdrawn_registration.date_retracted.isoformat().replace('+00:00', 'Z')
```

Comment:
Older API versions should have this date format `YYYY-MM-DDTHH:MM:SS.mmmmmm`, newer versions have `YYYY-MM-DDTHH:MM:SS.mmmmmmZ`